src/ontology/components/temporal-intervals.owl

```diff
@@ -10,7 +10,6 @@
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/elements/1.1/">
     <owl:Ontology rdf:about="http://purl.obolibrary.org/obo/ro/components/temporal-intervals.owl">
-        <owl:imports rdf:resource="http://purl.obolibrary.org/obo/ro/core.owl"/>
```

Comment:
This is an oversight from the #492 PR sorry about that. Couldnt refresh imports w/o fixing this.